README.md

```diff
@@ -1,17 +1,215 @@
-# iis
+# IIS
```

Comment:
I believe @jbondpdx and group want this to be lowercase to signify this is the module, not the technology.